src/utils/config_filters.hpp

```diff
@@ -69,4 +72,11 @@ bool string_matches_if_present(
 
 bool bool_or_empty(const config& filter, const config& cfg, const std::string& attribute);
 
+/**
+ * vector used for verify what attribute checked can be present in special/abilities.
+ */
+const std::set<std::string> abilities_tags{"tailwind", "hides", "skirmisher", "teleport", "leadership", "illuminates", "heals", "regenerate"};
```

Comment:
Needing to put a campaign-specific ability (`tailwind`) in this list suggests that it shouldn't be engine-level.